enterprise_access/apps/subsidy_access_policy/content_metadata_api.py

```diff
@@ -111,21 +111,36 @@ def get_list_price_for_content(enterprise_customer_uuid, content_key, content_me
             raise ContentPriceNullException(f'Could not determine list price for {content_key}') from exc
 
     # Note that the "content_price" key is guaranteed to exist, but the value may be None.
-    return list_price_dict_from_usd_cents(content_metadata['content_price'])
+    return make_list_price_dict(integer_cents=content_metadata['content_price'])
 
 
-def list_price_dict_from_usd_cents(list_price_integer_cents):
+def make_list_price_dict(decimal_dollars=None, integer_cents=None):
```

Comment:
nit: maybe it's worth doing some type validation on these? for example, so that we don't accidentally add code like `make_list_price_dict(decimal_dollars=1000)` when we really mean `make_list_price_dict(integer_cents=1000)`.

Reply:
Hm, not sure how one would differentiate since e.g. 1000 dollars is a believable price.

Reply:
Yeah, I just mean in the sense that it's not a believable `Decimal`. Maybe adding
```python
assert (integer_cents is None or isinstance(integer_cents, int))
assert (decimal_dollars is None or isinstance(decimal_dollars, (Decimal, float)))
```
...something like that, just as a foot-gun defense.